update based on new responder events

aidenkeating opened the pull request:
- show incident details on mission tab
- react to responder create and delete events

verification:
- update the emergency response buildconfig to point to this repo/branch
- create a new build
- create some responders using the disaster simulator, ensure they are shown in the dashboard without a refresh
- click the clear responders button at the bottom of the disaster simulator, ensure they are removed in the dashboard without a refresh

btison commented:
@aidenkeating 
Hi Aiden, this seems to work fine in general. However, I noticed that especially when creating a large number of responders at once (in my tests > 200), a couple of 404 error occur when calling the responder service endpoint `/responder/<id>`. I believe the reason for this is that on the responder service the Postgresql transaction is not fully committed yet when the endpoint is called.
To avoid this, can you check for a 404, and retry, potentially after a short pause ?

aidenkeating commented:
@btison I've added 7201763 to do this.

It'll try 5 times by default with an increased backoff each time.

I've also updated the way the updates work so that updates show in the UI so that if you add `500` responders you'll see each one being added to the total responders, instead of them all being dumped as a single UI update that just increases the total responders by 500. Happy to revert this change though if needed.
